src/infra/workload/releaseunit/variables-e2e.tfvars

```diff
@@ -1,7 +1,7 @@
 # Variable file for E2E env
 
 aks_node_size                   = "Standard_F8s_v2" # be aware of the disk size requirement for emphemral disks. Thus we currently cannot use a smaller SKU
-aks_node_pool_autoscale_minimum = 1
+aks_node_pool_autoscale_minimum = 2                 # 2 Nodes (of this SKU) is the minimum we need in order to run our workload
```

Comment:
Probably just a matter of preference, but I really don't like this spacing for comments :)

Reply:
its what `terraform fmt` does. so I'd say thats the way to go ;)

Reply:
That explains it, I hate what terraform fmt does :D